cpp/include/cudf/detail/utilities/vector_factories.hpp

```diff
@@ -28,6 +28,8 @@
 #include <rmm/device_uvector.hpp>
 #include <rmm/mr/device/device_memory_resource.hpp>
 
+#include <vector>
```

Comment:
is this needed?

Reply:
There are functions in this header that create and return `std::vector`.
The `vector_factories.hpp` should not rely on other files to include dependent headers for it.
I hit a compile error for this on some prototype test code that included `vector_factories.hpp` but did not need the `std::vector` functions.